samples/Clojure/validate-and-format.bb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bb
```

Comment:
The other test failure is because of this line. There are two solutions:

1. Replace the sample with one without a shebang (I see this doesn't appear to be a requirement)
2. Add the following to the Clojure entry in `languages.yml`:

```yaml
interpreters:
- bb
```

Reply:
Thanks for the help!